test/testcafe/spec/uiDemo_spec.js

```diff
@@ -0,0 +1,39 @@
+import { RequestMock } from 'testcafe';
+import UiDemoPageObject from '../framework/page-objects/UiDemoPageObject';
+import uiDemoResponse from '../../../playground/mocks/data/idp/idx/_ui-demo.json';
+
+const uiDemoMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(uiDemoResponse);
+
+async function setup(t) {
+  const pageObject = new UiDemoPageObject(t);
+  await pageObject.navigateToPage();
+  await t.expect(pageObject.formExists()).eql(true);
+  return pageObject;
+}
+
+fixture('UI demo');
+
+test
+  .requestHooks(uiDemoMock)('UI demo VRT', async t => {
```

Comment:
Should this be marked only to run in Gen3?

Reply:
this fixture is now disabled from `testcafe` and `parity-v3` above